R/draw-dm.R

```diff
@@ -1,20 +1,20 @@
-#' Draw schema of a [`dm`]-object's data model
+#' Draw a schema of a [`dm`]-object's data model
```

Comment:
schema -> diagram?

Reply:
You mention later on that the "data model part" has been removed from `dm`. Would that change drawing the diagram?